Updates to accessility section (#830)
carpentries/workshop-template: index.md

{% endcomment %}
<p id="accessibility">
  <strong>Accessibility:</strong>
  We are committed to making this workshop

>   accessible to everybody.  For workshops at a physical location, the workshop organizers have checked that:

Reviewer: ```suggestion
  accessible to everybody. 
{% if online == "false" %}
  The workshop organizers have checked that:
```

<p>
  <ul>
    <li>The room is wheelchair / scooter accessible.</li>
    <li>Accessible restrooms are available.</li>

>   </ul>

Reviewer: We need this part to still be captured within an `if`, as it should only be displayed for in-person events.

Reviewer: ```suggestion
  <ul>
    <li>The room is wheelchair / scooter accessible.</li>
    <li>Accessible restrooms are available.</li>
  </ul>
{% endif %}
```
